doc/Command_Index_gregorio.tex

```diff
@@ -751,9 +751,6 @@ \section{Gregorio Controls}
   \#4 & \TeX\ code & Macros rendering the first syllable; should emit the initial and populate \verb=\gre@opening@syllabletext=.\\
 \end{argtable}
 
-\macroname{\textbackslash GreScoreReference}{\#1}{gregoriotex-main.tex}
```

Comment:
Ok, then the mismatch is on the C side.  The C code still generates this.  I'll create an issue to take care of that.